ui/lib/core/addon/components/form-field.hbs

```diff
@@ -20,22 +20,42 @@
     {{/unless}}
   {{/unless}}
   {{#if @attr.options.possibleValues}}
-    <div class="control is-expanded">
-      <div class="select is-fullwidth">
-        <select name={{@attr.name}} id={{@attr.name}} onchange={{this.onChangeWithEvent}} data-test-input={{@attr.name}}>
-          {{#if @attr.options.noDefault}}
-            <option value="">
-              Select one
-            </option>
-          {{/if}}
-          {{#each (path-or-array @attr.options.possibleValues @model) as |val|}}
-            <option selected={{eq (get @model this.valuePath) (or val.value val)}} value={{or val.value val}}>
-              {{or val.displayName val}}
-            </option>
-          {{/each}}
-        </select>
+    {{#if (eq @attr.options.editType "radio")}}
+      <div class="control columns is-gapless" data-test-radio-input>
+        {{#each (path-or-array @attr.options.possibleValues @model) as |val|}}
+          <div class="column is-1 is-flex-center has-text-grey">
+            <RadioButton
+              class="radio"
+              name={{@attr.name}}
+              id={{or val.value val}}
```

Comment:
should we `dasherize` this value in case it's a value with spaces? (We would have to do the same for the label's `for` attr too)